java/code/src/com/suse/manager/reactor/hardware/HardwareMapper.java

```diff
@@ -183,7 +183,10 @@ else if (CpuArchUtil.isAarch64(cpuarch)) {
         CPUArch arch = ServerFactory.lookupCPUArchByName(cpuarch);
         cpu.setArch(arch);
 
-        cpu.setNrsocket(grains.getValueAsLong("cpusockets").orElse(null));
+        cpu.setNrsocket(grains.getValueAsLong("cpu_sockets").orElse(1L));
+        cpu.setNrCore(grains.getValueAsLong("cpu_core").orElse(
+                grains.getValueAsLong("total_num_cpus").orElse(0L)));
```

Comment:
do `cpu_core` and `total_num_cpus` contain the same thing or is there any logical difference?

Reply:
I cannot remember why I did that. I will change this